### ISSUE TITLE
[browserCapabilities] Opera always gets 'es5-bundled'

### ISSUE THREAD

**Issue by exunreal** [Polymer/tools#640]
I noticed that despite that Opera supports es2015 and push it always loads es5-bunled.
Example: https://shop.polymer-project.org

I found that the reason for that is in browserCapabilities.

In `browser-capabilities.js` there is a part with switch-case:

```javascript
    'Chrome': chrome,
    'Chromium': chrome,
    'Chrome Headless': chrome,
    'OPR': {
        es2015: since(36),
        es2016: since(45),
        es2017: since(45),
        es2018: since(51),
        push: since(28),
        serviceworker: since(32),
        modules: since(48),
    },
```

Here `OPR` has to be replaced with `Opera` cause ua-parser-js never returns 'OPR' as broserName for any version of Opera.

P.S. I've tested on Opera 54.

**Comment by aomarks** [comment]
Thanks for reporting, a PR would be welcome! :)

**Comment by aomarks** [comment]
Released in 1.1.2

**Comment by TimvdLippe** [comment]
@aomarks Would it be possible to issue a new CLI release to pick this fix up? Users are facing this issue with `polymer serve`: https://github.com/Polymer/polymer/issues/5353#issuecomment-419098396
